src/react-components/avatar-preview.js

```diff
@@ -63,6 +63,26 @@ function fitBoxInFrustum(camera, box, center, margin = DEFAULT_MARGIN) {
   camera.lookAt(center);
 }
 
+function getThemeBackground() {
+  const currentTheme = APP.store.state.preferences.theme;
+  const themes = window.APP_CONFIG?.theme.themes;
+  const defaultColor = 0xeaeaea;
+  if (currentTheme === "hubs-default") {
+    return defaultColor;
+  }
+  for (let i = 0; i < themes.length; i++) {
+    if (themes[i].id === currentTheme) {
+      let bgHex = themes[i].variables["background3-color"];
+      bgHex = `0x${bgHex.substring(1)}`;
+      if (bgHex.length !== 8) {
```

Comment:
Not sure if we intend people to use any type of CSS color value when defining their themes, but if we wanted to support other variations of CSS colors, we could use THREE.Color to do the conversion for us, instead of assuming hex values here.

Reply:
made a new commit taking all the above feedback into account. Thank you for all the great suggestions and questions.